x/qgb/keeper/keeper_valset.go

```diff
@@ -125,15 +123,18 @@ func (k Keeper) GetCurrentValset(ctx sdk.Context) (types.Valset, error) {
 }
 
 // normalizeValidatorPower scales rawPower with respect to totalValidatorPower
-// to take a value between 0 and 2^32 Uses BigInt operations to avoid overflow
-// errors Example: rawPower = max (2^63 - 1), totalValidatorPower = 1 validator:
-// (2^63 - 1)
+// to take a value between 0 and 2^32.
+// Uses BigInt operations to avoid overflow errors.
+// Example: rawPower = max (2^63 - 1), totalValidatorPower = 1 validator: (2^63 - 1)
```

Comment:
[nit] Is a comma missing here?
```suggestion
// Example: rawPower = max (2^63 - 1), totalValidatorPower = 1, validator: (2^63 - 1)
```
The clarity of the description can be improved e.g., it is not immediate that there is only one validator (if I understood it correctly)

Reply:
Yes, could be improved. This comment was copied from the gravity bridge, that's why I left it the way it is and only tried to reformat it